docs/cndb_omf.md

```diff
@@ -0,0 +1,294 @@
+DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!!
+
+# OMF descriptions
+
+Describes the proposed CNDB omf for HSE-3.0.
+
+## Record types
+
+Each record type contains a header at the beginning. This header contains the type of the record and its
+size. This header is not listed in the following breakdown of the various record types.
```

Comment:
Record type names are inconsistent and quite non-descriptive.  For example, ACK-N pairs with TXNODE, but ACK-C pairs with TXC.  How about we migrate to some variation of this:
```
   INFO   --> KVS_CREATE
   INFOD  --> KVS_DELETE
   TX     --> TX_START
   TXNODE --> TX_NODE
   TXC    --> TX_KVSET_CREATE
   TXM    --> TX_KVSET_META
   TXD    --> TX_KVSET_DELETE
   ACK    --> ACK
   NAK    --> NACK
```